import java.io.IOException;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

//test
public class Main{
	public static void main(String[] args) {
		
		/*
		Customer cust = new Customer();
		
		System.out.println(cust);
		System.out.println("Hello World!");
		System.out.println("Hello World!");
		*/
		
		
		
		
		/* TESTING (CAN BE DELETED) 
		Pizza testPizza = new Pizza();
		
		Map.Entry<Integer, ArrayList<Pizza>> testEntry = Map.entry(1, new ArrayList<Pizza>());
		testEntry.getValue().add(testPizza);
		
		
		try {
			Orders.checkForOrderFile();
			Orders.addOrderToFile(testEntry);
		} catch (IOException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
		*/
	}
}